--- apparmor-2.9.0/changehat/mod_apparmor/Makefile~	2014-09-15 20:30:47.000000000 +0200
+++ apparmor-2.9.0/changehat/mod_apparmor/Makefile	2014-10-29 22:49:19.303255857 +0100
@@ -57,9 +57,9 @@
   endif # LIBAPPARMOR not set
   LDLIBS += $(LIBAPPARMOR)
 else
-  LIBAPPARMOR_SRC := ../../libraries/libapparmor/
+  LIBAPPARMOR_SRC := ../../libraries/libapparmor
   LIBAPPARMOR_INCLUDE = $(LIBAPPARMOR_SRC)/include
-  LIBAPPARMOR_PATH := $(LIBAPPARMOR_SRC)/src/.libs/
+  LIBAPPARMOR_PATH := $(LIBAPPARMOR_SRC)/src/.libs
     ifeq ($(realpath $(LIBAPPARMOR_PATH)/libapparmor.a),)
         ERROR_MESSAGE = $(error ${nl}\
 ************************************************************************${nl}\
